README.md

```diff
@@ -315,6 +315,7 @@ You don't have to do anything manually!
 1. zonomi.com DNS API
 1. DreamHost.com API
 1. DirectAdmin API
+1. Zilore API
```

Comment:
you lost `kinghost`  here.

Reply:
I don't know how to edit latest versions, not stale ones - resolving this